### ISSUE TITLE
Getting error when using img2img mode (text2img works fine) 

### ISSUE THREAD

**Issue by JeffreyBull76** [Mikubill/sd-webui-controlnet#135]
Error running process: D:\StableDMain\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py
Traceback (most recent call last):
  File "D:\StableDMain\stable-diffusion-webui\modules\scripts.py", line 386, in process
    script.process(p, *script_args)
  File "D:\StableDMain\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 327, in process
    raise RuntimeError(f"model not found: {model}")
RuntimeError: model not found:

**Comment by JeffreyBull76** [comment]
Forgive my ignorance I am a total novice with Python, do you mean the full output in the cmd window or is there a full error log posted / generated somewhere I can access?

**Comment by JeffreyBull76** [comment]
In case you mean the former this is the full output...

`restoring last networks
Error running process: D:\StableDMain\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py
Traceback (most recent call last):
  File "D:\StableDMain\stable-diffusion-webui\modules\scripts.py", line 386, in process
    script.process(p, *script_args)
  File "D:\StableDMain\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 385, in process
    raise RuntimeError(f"model not found: {model}")
RuntimeError: model not found:

100%|██████████████████████████████████████████████████████████████████████████████████| 11/11 [00:27<00:00,  2.47s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 11/11 [00:26<00:00,  2.38s/it]`

There is nothing else, it then runs the generation (as you can see) but without applying the model (so it just runs as standard img2img without controlNet) As I say text2img works absolutely fine

**Comment by wzol** [comment]
Same here:

Error running process: I:\_progs\StableDiffusionWebUI\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py
Traceback (most recent call last):
  File "I:\_progs\StableDiffusionWebUI\stable-diffusion-webui\modules\scripts.py", line 386, in process
    script.process(p, *script_args)
  File "I:\_progs\StableDiffusionWebUI\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 428, in process
    raise RuntimeError(f"model not found: {model}")
RuntimeError: model not found: None

**Comment by JeffreyBull76** [comment]
As an update something completely separate went wrong with my Automatic 1111 install so I did a fresh install. And it works fine now. It's possible this might be related to where you have Automatic installed. My original one was installed on my D: drive this time I did it on my main C: drive and its fine. Works perfectly now. Might not be related at all might have just been a bad install the first time or a version issue? But just thought I'd post in case it helps isolate the original issue.
